src/components/Error.tsx

```
@@ -0,0 +1,33 @@
+import React, { memo } from 'react'
+
+import { ErrorBoundaryFallbackComponent } from '@/src/components/ErrorBoundary'
+import { useIsomorphicEffect } from '@/src/hooks/useIsomorphicEffect'
+import { enqueSnackbar } from '@/src/redux/snackbarSlice'
+import { dispatch } from '@/src/redux/store'
+
+interface Props<T extends Error> {
+  error?: T | FetchBaseQueryError | SerializedError | undefined
+}
+
+export const Error: React.FC<React.PropsWithChildren<Props<any>>> = memo(
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)